CIPx/CIP-xxxx.md

```diff
@@ -0,0 +1,103 @@
+Title: On-chain stake pool operator to delegates communication
+Authors: Marek Mahut <marek.mahut@fivebinaries.com>, Sebastien Guillemot <sebastien@emurgo.io>, Ján Hrnko <jan.hrnko@fivebinaries.com>
+Comments-URI: https://forum.cardano.org/t/on-chain-stake-pool-operator-to-delegates-communication/42229
+Status: Draft
+Type: Standards
+Created: 2020-11-07
+License: CC-BY-4.0
+Requires: CIP10
+---
+
+## Abstract
+
+Standard format for metadata used in on-chain stake pool owner communication towards their delegates.
+
+## Terminology
+
+
+We define two types of communication metadata, which are distinguished by the transaction metadata label as defined in the [CIP10 Transaction metadata label registry](https://github.com/cardano-foundation/CIPs/blob/master/CIP10/README.md).
+
+ * *Message board communication* is a type of metadata that has been included in a on-chain transaction between two base addresses associated with the stake pool operator owner address. Given the onetime fee for this communication, we are considering this as a message board of the stake pool, as it also enables delegates to easier access historical metadata communication.
+
+ * *Direct delegate communication* is a type of metadata that has been included in an on-chain transaction between the stake pool owner account and the delegate's account. This type of communication is more expensive for the stake pool owner, preventing higher abuse and therefore enables wallets to implement notification granularity. It might be suitable for targeting specific delegates, such as messaging only new joined delegates, loyal delegates, high-amount delegates etc.
+
+
+
+## Motivation
+
+The lack of on-chain communication standard between the stake pool owner and their delegates.
+
+<!-- Link to CIP6 link once/if merged -->
+
+Working in progrress [CIP6](https://github.com/cardano-foundation/CIPs/pull/15) defines an external feed of a stake pool within the extended metadata, but as an alternative, there is a need for more verifiable on-chain communication standard, that will also ensure associated cost with such communication to prevent its abuse.
+
+## Specification
+
+As per [CIP10 Transaction metadata label registry](https://github.com/cardano-foundation/CIPs/blob/master/CIP10/README.md), we assign:
+
+* *Message board communication* the transaction metadata label `1990`,
+* *Direct delegate communication* the transaction metadata label `1991`.
+
+## Metadata
+
+Metadata is written in a JSON format and the maximum size of the metadata is 16kb.
+
+| key | Value | Rules |
+| --- | ---  | --- |
+| `title` *(required)*| Title of the communication | 64 bytes UTF-8 encoded string  |
+| `content` *(required)*| Content of the communication | An array of 64 bytes UTF-8 encoded strings |
+|||
+| `language` *(optional)*| ISO 639-3 languange code of the content | 3 bytes UTF-8 encoded string
+| `link` *(optional)*| Link for additional communication | 64 bytes UTF-8 encoded string, must be a valid URL |
```

Comment:
May I suggest to _not_ add any link to avoid phishing attempts, and keep the message as simple as possible. 
The rationale being that delegators should know where to find information about their pools if needed and need not to click on anything.

Reply:
As per today's meeting, removing the link field in https://github.com/cardano-foundation/CIPs/pull/44/commits/5ad3477ee9dfb79238f01163ac29b40c5ddd102e.